pixano/data/importers/coco_importer.py

```diff
@@ -53,12 +55,16 @@ def __init__(
         tables = super().create_tables(
             media_fields,
             object_fields={
+                "original_id": "str",
                 "bbox": "bbox",
                 "mask": "compressedrle",
                 "category": "str",
             },
         )
 
+        # Add original id in main table
+        tables["main"][0].fields["original_id"] = "str"
```

Comment:
Maybe add "original_id" to base importer (importer.py, line 104), instead of adding it in each importers (COCO, DOTA, Image) manually.